src/server.ts

```diff
@@ -201,92 +235,128 @@ app.post("/callAnswered", async (req, res) => {
   let data: CallData = {
     from: req.body.from,
     to: req.body.to,
-    bridge: false,
+    callType: "outbound",
   };
 
   voiceCalls.set(callId, data); // preserve the info on the bridge leg in the calls map.
 
-  let bridgecallId: string = "";
-  for (let [key, value] of voiceCalls.entries()) {
-    if (value.bridge) {
-      bridgecallId = key;
-    }
-  }
   // This is the response payload that we will send back to the Voice API to conference the call into the WebRTC session
-
+  var response = new Response();
   var speakSentence = new SpeakSentence({
-    sentence: "The bridge will start now",
-  });
-
-  var bridge = new Bridge({
-    callId: bridgecallId,
-    // bridgeCompleteUrl: `${voiceCallbackUrl}/callStatus`,
-    bridgeTargetCompleteUrl: `${voiceCallbackUrl}/bridgeTargetComplete`,
+    sentence: "The call will start now",
   });
-
-  var response = new Response();
   response.add(speakSentence);
-  response.add(bridge);
 
-  let myResp = await response.toBxml();
-  console.log(myResp);
+  const bridgecallId: string = findCall("bridge");
+  if (bridgecallId) {
+    // the sipinterconnect call exists - bridge them
+    var bridge = new Bridge({
```

Comment:
Global change to make... Try to replace var with either const or let. Usage of var is discouraged

Reply:
bad Steve :-)   I'm going to blame my bad habit of clone and modify :-(  Thanks for the reminder.